tools/agent-ctl/Makefile

```diff
@@ -6,7 +6,7 @@
 default: build
 
 build:
-	cargo build -v
+	RUSTFLAGS="--deny warnings" cargo build -v
```

Comment:
ideally we should use `cargo clippy` to check all these warnings. Something for future improvements.